### ISSUE TITLE
Generate a circosplot for a bacterial genome + map other stuffs

### ISSUE THREAD

**Issue by BioRB** [lvulliard/BioCircos.R#24]
Dear Developers,
I'm trying to use your script to map methylation positions on a bacterial genome of 4M bp, but something is going wrong because in the end I obtain an empty plot.
Here what I do:
`library(BioCircos)
myGenome = list ("Bacterial" = 4000000)
BioCircos(genome = myGenome, genomeFillColor = c("Reds"), chrPad = 0, genomeLabelDy = 25, genomeTicksScale = 4e+4)
csv <- read.csv("file.csv", sep="\t", header = FALSE)  
points_chromosomes = c("chromosome_1")
points_coordinates <- csv$V2   
points_values <- csv$V5
tracklist = BioCircosSNPTrack('mySNPTrack', points_chromosomes, points_coordinates, 
  points_values, colors = c("tomato2", "tomato2"), minRadius = 0.5, maxRadius = 0.9)
BioCircos(tracklist, genome = myGenome, genomeFillColor= c("tomato2", "darkblue"), chrPad = 0.02, displayGenomeBorder = FALSE, genomeTicksDisplay = FALSE,  genomeLabelTextSize = 0)
`
I get an empty orange circle in the end. no methylation circle inside. no methylation positions.
the 2 column that i use for points_coordinates and points_values  look like this: 
![Screenshot from 2022-11-02 15-10-30strap](https://user-images.githubusercontent.com/43418626/199512812-d0fe46de-20f0-472f-a3ca-cbd2a551f0e6.png)

basically I need 1 circle reppresening the bacterial genome and another inside for the methylation. I have a file with 2 columns, 1 for the genomic position and the other for the level of methylation.  
thank a lot!
Best,
RB
